File: Shared/Templates/Application/ObjC/ViewController.m
#import "ViewController.h"

@implementation ViewController

- (void)viewDidLoad {
	[super viewDidLoad];

	self.view.backgroundColor = [UIColor systemBackgroundColor];

	UIImageSymbolConfiguration *config = [UIImageSymbolConfiguration configurationWithScale:UIImageSymbolScaleLarge];
	UIImage *img = [[UIImage systemImageNamed:@"globe"] imageByApplyingSymbolConfiguration:config];
	UIImageView * globeView = [[UIImageView alloc] initWithImage:img];
	globeView.tintColor = [UIColor systemBlueColor];

	UILabel *label = [[UILabel alloc] init];
	label.text = @"Hello, world!";
	label.textAlignment = NSTextAlignmentCenter;
	label.textColor = [UIColor labelColor];

	UIStackView *stackView = [[UIStackView alloc] initWithArrangedSubviews:@[globeView, label]];
	stackView.axis = UILayoutConstraintAxisVertical;
	stackView.distribution = UIStackViewDistributionEqualCentering;
	stackView.alignment = UIStackViewAlignmentCenter;
	[self.view addSubview:stackView];
	stackView.translatesAutoresizingMaskIntoConstraints = NO;

	[NSLayoutConstraint activateConstraints:@[
		[stackView.centerXAnchor constraintEqualToAnchor:self.view.centerXAnchor],
		[stackView.centerYAnchor constraintEqualToAnchor:self.view.centerYAnchor]
	]];
}


@end

File: Shared/Templates/Application/Swift/UIKit/ViewController.swift
import UIKit

class ViewController: UIViewController {
	override func viewDidLoad() {
		super.viewDidLoad()
		
		view.backgroundColor = .systemBackground
		
		let config = UIImage.SymbolConfiguration(scale: .large)
		let imageView = UIImageView(image: UIImage(systemName: "globe", withConfiguration: config))
		imageView.tintColor = .systemBlue
		
		let label = UILabel()
		label.text = "Hello, world!"
		label.textAlignment = .center
		label.textColor = .label
		
		let stackView = UIStackView(arrangedSubviews: [imageView, label])
		stackView.axis = .vertical
		stackView.distribution = .equalCentering
		stackView.alignment = .center
		stackView.translatesAutoresizingMaskIntoConstraints = false
		view.addSubview(stackView)
		
		NSLayoutConstraint.activate([
			stackView.centerXAnchor.constraint(equalTo: view.centerXAnchor),
			stackView.centerYAnchor.constraint(equalTo: view.centerYAnchor)
		])
	}
}
